src/Message/Response.php

```diff
@@ -65,6 +68,17 @@ public function getMessage()
         return '';
     }
 
+    public function isPending(): bool
```

Comment:
It looks like it should do the work, but I would like to see some unit tests manually constructing few types of responses and asserting that they are considered pending